en/mkdocs.yml

```diff
@@ -1056,6 +1058,6 @@ extra:
       version: 1.2.0-1
       git_tag: v1.2.0.1
   site_version: 4.3.0
-  #base_path: http://localhost:8000/en/latest
+  # base_path: http://localhost:8000/en/latest
```

Comment:
Lets not change this

Reply:
Reverted by https://github.com/wso2/docs-mi/pull/1530/commits/1c7ee73efc8e83197c8be971e61526ef307447cd